packages/ui/tsup.config.web.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  entry: ['web.ts'],
+  format: ['esm'],
+  tsconfig: 'tsconfig.web.json',
+  outDir: 'dist-web/',
```

Comment:
Needs `dts: true` no?

Reply:
I added this and some other options with inspiration from https://github.com/fungible-systems/micro-stacks/blob/main/packages/core/tsup.config.ts. 

Thanks!